bin/desi_tile_vi

```diff
@@ -189,15 +186,13 @@ def main():
         selection &= (tiles_table["EFFTIME_SPEC"]>=(tiles_table["GOALTIME"]*tiles_table["MINTFRAC"]))
         selection &= ((tiles_table["QA"]!="good")&(tiles_table["QA"]!="bad") |
                       (tiles_table["QA"]=="bad")&(tiles_table["LASTNIGHT"]>tiles_table["QANIGHT"]))
-    if args.new :
-        selection &= (tiles_table["QA"]=="none")
-    if args.survey is not None :
-        selection &= (tiles_table["SURVEY"]==args.survey)
-    if args.program is not None:
-        selection &= np.isin([x.lower() for x in tiles_table['FAPRGRM']],
-                             [x.lower() for x in args.program])
-    if args.qastatus is not None:
-        selection &= (tiles_table["QA"]==args.qastatus)
+        if args.survey is not None:
+            selection &= (tiles_table["SURVEY"]==args.survey)
+        if args.program is not None:
+            selection &= np.isin([x.lower() for x in tiles_table['FAPRGRM']],
+                                 [x.lower() for x in args.program])
+        if args.qastatus is not None:
```

Comment:
This test implies that `args.qastatus` could be None, which I think could never happen since the default value is not None (though it is "none", meaning new tiles without a QA status, which is confusingly similar to `None`).  Consider dropping the if test or at least adding a comment that this is a belt-and-suspenders check just in case the default later changes to `None` instead of `"none"`.

Bonus points: Since this is an opportunity for the user to supply an invalid value that would filter out all rows without it being obvious why; consider adding a safety check like
```python
keep = (tiles_table["QA"] == args.qastatus)
if np.any(keep):
    selection &= keep
else:
    print(f'ERROR: no tiles with QA={args.qastatus}; valid values are {set(tiles_table["QA"])}')
    sys.exit(1)
```